src/Program.fs

```diff
@@ -400,10 +407,29 @@ let generate (configFile: string) =
                 colorprintfn "✏️  Generating Fable project $green[%s]" projectPath
                 let files =
                     generatedFiles
-                    |> Seq.map (fun file -> sprintf "        <Compile Include=\"%s\" />" (Path.GetFileName file))
-                    |> String.concat "\n"
-
-                File.WriteAllText(projectPath, CodeGen.sampleFableProject files)
+                    |> Seq.map
+                        (fun file ->
+                            XElement.ofStringName("Compile",
+                                XAttribute.ofStringName("Include", file |> Path.GetFileName)
+                        ))
+                let packageReferences =
+                    packageReferences
+                    |> List.append
+                        (seq {
+                            XElement.PackageReference("Fable.SimpleHttp", "3.0.0")
+                            XElement.PackageReference("Fable.SimpleJson", "3.19.0")
+                        } |> Seq.toList)
```

Comment:
Can you please refactor this piece of code to remove the noise around `seq` and `list`?
```fs
let packageReferences = 
      packageReferences
      |> List.append [
          XElement.PackageReference("Fable.SimpleHttp", "3.0.0")
          XElement.PackageReference("Fable.SimpleJson", "3.19.0")
      ]
```

Reply:
Maybe
``` F#
[ yield! packagePreferences
  yield XElement.PackageReference("Fable.SimpleHttp", "3.0.0")
  yield XElement.PackageReference("Fable.SimpleJson", "3.19.0") ]
```

Reply:
Or
``` F#
XElement.PackageReference("Fable.SimpleHttp", "3.0.0") ::
XElement.PackageReference("Fable.SimpleJson", "3.19.0") ::
packageReferences
```